de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/spi/InvoiceCandidateListenerAdapter.java

```diff
@@ -32,22 +32,28 @@
 
 /**
  * Abstract implementation of {@link IInvoiceCandidateListener} which does nothing.
- * 
+ *
  * This class can be used as a template for implementing particular {@link IInvoiceCandidateListener}s.
- * 
+ *
  * @author tsa
  *
  */
 public abstract class InvoiceCandidateListenerAdapter implements IInvoiceCandidateListener
 {
 
+	@Override
+	public void onBeforeClosed(I_C_Invoice_Candidate candidate)
```

Comment:
pls move it last, preserve the order from interface.

Reply:
ok